Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -26,7 +26,95 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
             this.on('*:changeLanguage', this._selectLanguage);
         },
 
+        /**
+         * Sets the language code for the content creation.
+         * Override the Content Edit View Service implementation to force the
+         * value to the default content creation language.
+         *
+         * @method _setLanguageCode
+         * @protected
+         */
+        _setLanguageCode: function () {
+            this.set('languageCode', this.get('app').get('contentCreationDefaultLanguageCode'));
+        },
+
         _load: function (next) {
+            var request = this.get('request'),
+                tasks = new Y.Parallel(),
+                hasError = false,
+                contentType = this.get('contentType');
+
+            if ( request.route.name === 'createContent' ) {
+                this._deprecatedLoad(next);
+                return;
+            }
+
+            contentType.set('id', request.params.contentTypeId);
+            contentType.load({api: this.get('capi')}, tasks.add(Y.bind(function (err) {
+                if ( err ) {
+                    hasError = true;
+                    this._error("Could not load the Content Type");
+                }
+            }, this)));
+
+            this._loadParentLocation(request.params.parentLocationId, tasks.add(Y.bind(function (err) {
+                if ( err ) {
+                    hasError = true;
+                    this._error("Could not load the parent Location and Content");
+                }
+            }, this)));
+
+            tasks.done(Y.bind(function () {
+                if ( !hasError ) {
```

Comment:
normally, you stay on the same page if an error happens at this stage. We could probably improve the error handling but that's a bit outside of the scope of this story/PR but feel free to open a improvement in JIRA with ideas and expected behavior.